packages/components/src/custom-select-control-v2/types.ts

```diff
@@ -70,9 +79,7 @@ export type _CustomSelectProps = CustomSelectButtonProps & {
 	label: string;
 };
 
-export type CustomSelectProps = _CustomSelectProps &
-	CustomSelectButtonProps &
```

Comment:
Removed `CustomSelectButtonProps` since it's already part of `_CustomSelectProps`

Reply:
@mirka — just making sure that you see this change, since you refactored these types last

Reply:
Good catch, thanks 🙏